sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractKeyPhrasesResultCollectionDebugView.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Debugger Proxy class for <see cref="ExtractKeyPhrasesResultCollection"/>.
+    /// </summary>
+    public class ExtractKeyPhrasesResultCollectionDebugView
+    {
+        private ExtractKeyPhrasesResultCollection BaseCollection { get; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractKeyPhrasesResultCollectionDebugView"/> class.
+        /// </summary>
+        /// <param name="collection"></param>
+        public ExtractKeyPhrasesResultCollectionDebugView(ExtractKeyPhrasesResultCollection collection)
```

Comment:
we can follow what https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/Internal/ResponseDebugView.cs is doing. We won't need to add the docstrings so no worries about later keeping them up to date